packages/opentelemetry-exporter-collector-grpc/test/CollectorTraceExporter.test.ts

```diff
@@ -159,7 +159,7 @@ const testCollectorExporter = (params: TestParams) =>
         const spyLoggerWarn = sinon.stub(diag, 'warn');
         collectorExporter = new CollectorTraceExporter({
           serviceName: 'basic-service',
-          url: address + '/v1/trace',
+          url: `http://${address}/v1/trace`,
```

Comment:
proto part of url should be ignored anyway so it should be fine to leave it off? Not blocking just makes me curious what the failure was.

Reply:
yea but this unit test is testing a warning for that